services/web/server/src/simcore_service_webserver/oas3/v0/components/schemas/fake.yml

```diff
@@ -0,0 +1,36 @@
+FakeEnveloped:
+  type: object
+  required:
+    - data
+    - error
+  properties:
+    data:
+      $ref: '#/FakeType'
+      nullable: true
+      default: null
+    error:
+      $ref: "./error.yml#/ErrorType"
+      nullable: true
+      default: null
+
+FakeType:
+  type: object
+  required:
+    - path_value
+    - query_value
+    - body_value
+  properties:
+    path_value:
+      type: string
+    query_value:
+      type: string
+    body_value:
+      type: object
+      additionalProperties:
+        type: string
+  example:
+    path_value: foo
+    query_value: bar
+    body_value:
+      key1: value1
+      key2: value2
```

Comment:
Is this in use by webserver? if this is for the check rest api call, we should really change the name because it is misleading.;)